tac/gui/dashboards/agent.py

```diff
@@ -83,7 +83,6 @@ def __init__(self, agent_name: str,
 
     def init(self):
         """Re-initiate the AgntDashboard."""
-        self.viz.delete_env(self.env_name)
```

Comment:
This line caused some problems on the Visdom server - we were trying to delete a non-existing environment.